src/modules/authorization/actions.spec.ts

```diff
@@ -0,0 +1,55 @@
+import {
+  AUTHORIZATION_FLOW_FAILURE,
+  AUTHORIZATION_FLOW_REQUEST,
+  AUTHORIZATION_FLOW_SUCCESS,
+  authorizationFlowFailure,
+  authorizationFlowRequest,
+  authorizationFlowSuccess
+} from './actions'
+import { Authorization, AuthorizationAction } from './types'
+
+describe('authorization flow actions', () => {
```

Comment:
What do you think about making `authorization` global? It would remove the authorization definition from all the tests.
```suggestion
let authorization: Authorization

describe('authorization flow actions', () => {
  beforeEach(() => {
    authorization = {} as Authorization
  })
```